probcal/utils/experiment_utils.py

```diff
@@ -17,6 +18,8 @@
 from probcal.models.discrete_regression_nn import DiscreteRegressionNN
 from probcal.utils.configs import TrainingConfig
 
+GLOBAL_DATA_DIR = 'data'
```

Comment:
I moved this global variable here. If we have more of these, I think a `common` directory would make sense. Might be overkill for now.